src/std-types/word_counter_project/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
```

Comment:
This does not need a whole "project", just a single `.rs` file like the other exercises.